deployment/roles/reverse/templates/apache/page/index_links.html.j2

```diff
@@ -0,0 +1,33 @@
+#jinja2: lstrip_blocks: True
+          <div class="panel panel-default col-md-12 container">
+            <a class="panel-header no-toggle" href="#">
+              <h2>Vitam UI - Liens</h2>
+            </a>
+            <div class="panel-body">
+              <p class="row" style="vertical-align: middle">
+                <div class="col-md-3">Métier</div>
+                <div class="col-md-9">
+                  <a href="https://{{ vitam_reverse_external_dns }}" class="btn btn-primary btn-xs" target="_blank">Vitam UI</a>
+                </div>
+              </p>
+              {% if groups['hosts_vitamui_consul_server'] | default([]) | length > 0 %}
+              <p class="row" style="vertical-align: middle">
+                <div class="col-md-3">Consul</div>
+                <div class="col-md-9">
+                  <a href="/ui/{{ vitam_site_name }}-ui/services" class="btn btn-primary btn-xs" target="_blank">Consul</a>
+                  <a href="https://{{ vitam_reverse_external_dns }}/v1/health/state/any?pretty" class="btn btn-primary btn-xs" target="_blank">Health</a>
+                </div>
+              </p>
+              {% endif %}
+              {% if mongo_express_enabled | default(false) | bool == true %}
```

Comment:
La variable mongo_express_enabled est définie pour chaque host du groups['hosts_vitamui_mongod'];, il faudrait donc le déplacer dans la boucle for du dessous et probablement appeler la variable via un hostvars.

Reply:
C'est bien troublant d'avoir des organisations différentes entre Vitam et Vitam UI pour un même paramétrage. Je corrige ça.